### ISSUE TITLE
[feature request] Send custom message when client send heartbeat

### ISSUE THREAD

**Issue by Anilople** [alibaba/Sentinel#2084]
## Issue Description

Type: *feature request*

### Describe what happened (or what feature you want)

When sentinel-transport send heartbeat to dashboard's Endpoint, it can send user's custom messages.

### Describe what you expected to happen

User can custom information like follow:

```properties
group = basic
app = order
```

When sentinel-transport send heartbeat, it can add those messages.

### Tell us your environment

Sentinel Version: 1.8.1

**Comment by cdfive** [comment]
Good idea, the client application may have different and custom information, which can be displayed and aggregated in dashboard UI.
